Privacy.md

```diff
@@ -0,0 +1,56 @@
+# BlitzMail Privacy Policy
+This is the developer's policy which is created to give details and informs you if your personal information is collected, used, disclosed and secured by the developer. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. <hr>
+
+## Permission and Access
+
+BlitzMail software requires the storage permission to function effectively and efficiently. Without this permission enabled, you might be unable to perform certain operations with the software.
```

Comment:
You can be specific here: This is needed when sharing files such as photos from other apps.